tests/test_redis_backend.py

```diff
@@ -126,3 +126,10 @@ def test_lookup(self):
             end=datetime.date(2014, 2, 5),
             buckets=['group1', 'group2'],
         ))
+
+    def test_lookup_no_values(self):
+        self.assertEqual([], self.backend.lookup(
+            ActivityTracker.PERIOD_MONTHLY,
+            start=datetime.date(2013, 10, 5),
+            end=datetime.date(2013, 2, 5),
```

Comment:
Force pushed in https://github.com/educreations/activity-tracker/commit/edd60f0b4b2cf07603d4e6d1374b38288e817d13